src/components/elements/query-image.tsx

```diff
@@ -32,7 +32,7 @@ const QueryImage = ({
     loading,
     width,
     ...props
-}: QueryImageProps): React.ReactNode => {
+}: QueryImageProps): JSX.Element => {
```

Comment:
```suggestion
}: QueryImageProps): ReactElement => {
```
`ReactElement` is a more appropriate type to return for components with no children. ie; `<QueryImage />`